babel.config.js

```diff
@@ -23,5 +23,6 @@ module.exports = {
     require('@babel/plugin-proposal-class-properties'),
     require('@babel/plugin-proposal-object-rest-spread'),
     ['@babel/plugin-transform-modules-commonjs', { allowTopLevelThis: true }],
+    [require('@babel/plugin-transform-runtime'), { regenerator: true }],
```

Comment:
Thank you. I will also do this line of change when fixing functional tests for Alerting Kibana plugin.
Just a reminder that `true` is the default value by for the option `regenerator`, so it can be omitted.

Reply:
Thanks. good to know!